packages/shared/prisma/schema.prisma

```diff
@@ -1214,3 +1218,108 @@ model TableViewPreset {
   @@unique([projectId, tableName, name])
   @@map("table_view_presets")
 }
+
+model Action {
+  id        String   @id @default(cuid())
+  createdAt DateTime @default(now()) @map("created_at")
+  updatedAt DateTime @default(now()) @updatedAt @map("updated_at")
+
+  projectId String  @map("project_id")
+  project   Project @relation(fields: [projectId], references: [id], onDelete: Cascade)
+
+  type ActionType
+
+  // Configuration specific to each action type
+  config Json // Structured JSON for different action types
+  // For WEBHOOK: { version: "1.0", url: "...", method: "POST", headers: {...}, secretId: "..." }
+  // For ANNOTATION_QUEUE: { version: "1.0", queueId: "..." }
+
+  // Relations
+  triggers        Automation[] // Changed to use the join table
```

Comment:
Let's remove the comment as it doesn't make sense looking forward. Took me some time to understand what is going on there as the statement didn't make it obvious.